src/openforms/contrib/kadaster/api/tests/test_api_map_search.py

```diff
@@ -0,0 +1,248 @@
+from unittest.mock import MagicMock, patch
+
+from django.test import override_settings
+
+import requests
+from rest_framework import status
+from rest_framework.reverse import reverse
+from rest_framework.test import APITestCase
+
+from openforms.submissions.tests.factories import SubmissionFactory
+from openforms.submissions.tests.mixins import SubmissionsMixin
+
+
+@override_settings(LANGUAGE_CODE="en")
+class MapSearchApiTests(SubmissionsMixin, APITestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.submission = SubmissionFactory.create()
```

Comment:
please always call the super method so you don't get unexpected behaviour when adding/removing mixins